ee/clickhouse/queries/breakdown_props.py

```diff
@@ -91,6 +101,22 @@ def get_breakdown_prop_values(
     )[0][0]
 
 
+def _to_value_expression(
+    breakdown_type: Union[Literal["event", "person", "cohort", "group"], None],
+    breakdown: Union[str, List[Union[str, int]], None],
```

Comment:
Can breakdown be a single `int` as well?

Reply:
The breakdown value is boxed on the way in to funnel processing